### PR TITLE
updating readmes to include new info about the dashboards and data-bites

### PR DESCRIPTION
**Pull request by jeffrainey**

### CODE REVIEW

File: packages/chart/README.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ export default App;
 
 Note, you must pass in a configuration object. Configuration objects can be created with the [Editor package](https://github.com/CDCgov/cdc-open-viz/tree/main/packages/editor).
 
+```
+sampleConfig = {
```
Comment (danielimmke) [inline packages/chart/README.md:30]: This is not the right type of sample config. newViz: true is an internal flag. Recommend just link to a file in the /examples/ folder.

File: packages/dashboard/README.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ export default App;
 
 Note, you must pass in a configuration object. Configuration objects can be created with the [Editor package](https://github.com/CDCgov/cdc-open-viz/tree/main/packages/editor).
 
+```
+sampleConfig = {
```
Comment (danielimmke) [inline packages/dashboard/README.md:30]: This is not the right type of sample config. newViz: true is an internal flag. Recommend just link to a file in the /examples/ folder.

File: packages/data-bite/README.md [hunk 3]
```diff
@@ -26,6 +26,13 @@ export default App;
 
 Note, you must pass in a configuration object. Configuration objects can be created with the [Editor package](https://github.com/CDCgov/cdc-open-viz/tree/main/packages/editor).
 
+```
+sampleConfig = {
```
Comment (danielimmke) [inline packages/data-bite/README.md:30]: Again, probably can link to the /examples/ file here

**Review by danielimmke — CHANGES_REQUESTED** [review]
Hey Jeff, thanks for this. Just a few changes

File: packages/chart/README.md (hunk 1 above)
Comment (jeffrainey) [reply packages/chart/README.md]: This has been updated.

File: packages/dashboard/README.md (hunk 2 above)
Comment (jeffrainey) [reply packages/dashboard/README.md]: This has been updated.

File: packages/data-bite/README.md (hunk 3 above)
Comment (jeffrainey) [reply packages/data-bite/README.md]: This has been updated.

**Review by danielimmke — APPROVED** [review]

**Pull request merged**